Dockerfile

```diff
@@ -1,13 +0,0 @@
-FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
-WORKDIR /go/src/github.com/openshift/local-storage-operator
-COPY . .
-RUN make build
-
-FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
-COPY --from=builder /go/src/github.com/openshift/local-storage-operator/_output/bin/local-storage-operator /usr/bin/
-COPY manifests /manifests
-ENTRYPOINT ["/usr/bin/local-storage-operator"]
-LABEL com.redhat.delivery.appregistry=true
-LABEL io.k8s.display-name="OpenShift local-storage-operator" \
-      io.k8s.description="This is a component of OpenShift and manages local volumes." \
-        maintainer="Hemant Kumar <hekumar@redhat.com>"
```

Comment:
When this repo is imported in ART pipeline, these Dockerfiles are expected. Deleting them will ART pipeline to break I think.

Reply:
Copied it to root. I will readjust the path of Dockerfiles after this PR gets merged.